Microsoft.Toolkit.Uwp.UI.Controls/InAppNotification/Styles/MSEdgeNotificationStyle.xaml

```diff
@@ -1,26 +1,34 @@
-﻿<ResourceDictionary
-    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation" 
-    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-    xmlns:local="using:Microsoft.Toolkit.Uwp.UI.Controls">
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
```

Comment:
Are these just style formatting? @azchohfi I know we have XAML Styler definitions, but I feel like we haven't run them across the repo, think we should have a similar thing to 'update headers' which can fix all the styling?

Reply:
Yes, all the changes are related to XAML styler.
The only change I've made is to name the `ContentPresenter` in the template.